test/unit/modules/isShopgateJSBridgeAvailableTest.js

```diff
@@ -0,0 +1,18 @@
+import chai from 'chai';
+import { isShopgateJSBridgeAvailable } from '../../../src/modules/isShopgateJSBridgeAvailable';
+
+const { expect } = chai;
+
+describe('isShopgateJSBridgeAvailable', () => {
+  it('should return false when JSBridge is not set', () => {
+    global.window = {};
+    expect(isShopgateJSBridgeAvailable()).equal(false);
+  });
+
+  it('should return true when JavascriptBridge is set', () => {
+    global.window = {};
+    global.window.SGJavascriptBridge = {};
```

Comment:
Would make sense to cover undefined and null.

Reply:
We talked about it. The SGJavaScriptBridge is set by our apps and when we would test for null and undefined - we should also test for values like an empty array or -1 etc. As long as this is no user input we are talking about here we shouldn't create that much overhead